trade-finance-manager-api/src/graphql/reducers/mappings/facilities/mapUkefExposure.js

```diff
@@ -24,6 +24,9 @@ const mapUkefExposure = async (facilityTfm, facility) => {
 
       if (isValidCompletedValueAmendment(latestCompletedAmendment)) {
         const { coverPercentage, coveredPercentage } = facility.facilitySnapshot;
+        const { requireUkefApproval, submittedAt, bankDecision } = latestCompletedAmendment;
+        // value of ukefExposureCalculationTime from automatic amendment submission time or manual amendment bankDecision submission time
+        const ukefExposureCalculationTime = requireUkefApproval ? bankDecision.submittedAt : submittedAt;
```

Comment:
Are saving precise time of `UKEF Exposure` calculation time?

Reply:
no, because it is done every time we map once amendment completed.  if no amendment then it's saved as deal submission date in mongodb

Reply:
Should we rename the variable to `submissionDate` or something similar (if I have understood the reasoning behind the variable).

Reply:
So, in tfm facilities-> tfm.  Without amendments the value is stored as ukefExposureCalculationTimestamp which is generated on deal submission and is epoch milliseconds.  When an amendment is submitted, there is submittedAt value which is generated using getUnixTime() from date-fns.  May need @IgnatG input if we are changing the name

Reply:
So, in tfm facilities-> tfm. Without amendments the value is stored as ukefExposureCalculationTimestamp which is generated on deal submission and is epoch milliseconds. When an amendment is submitted, there is submittedAt value which is generated using getUnixTime() from date-fns. May need @IgnatG input if we are changing the name

Reply:
done